src/Raven.Server/Utils/Metrics/Commands/GetDatabaseMetricsCommand.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Net.Http;
+using System.Text;
+using Raven.Client.Http;
+using Sparrow.Json;
+
+namespace Raven.Server.Utils.Metrics.Commands
+{
+    internal class GetDatabaseMetricsCommand : RavenCommand<BlittableJsonReaderObject>
```

Comment:
Avoid the `BlittableJsonReaderObject` here?

Reply:
done